recipes/arb-bio/6.1.rev16223/abs_path_fix.patch

```diff
@@ -0,0 +1,753 @@
+Index: BINDINGS/PERL/Makefile
+===================================================================
+--- BINDINGS/PERL/Makefile	(revision 16223)
++++ BINDINGS/PERL/Makefile	(working copy)
```

Comment:
I probably could, but the hard coded /usr/bin/perl paths are correct for other scenarios. ARB comes with Perl language bindings and uses them from its scripts, so the Perl used by those has to match the one used during build.

In Bioconda, the package requires and pins Perl, so the match is guaranteed to be in the PATH as long as ARB is.

In e.g. Debian however, the package would be built against the system Perl, and the scripts need to use that one even if the user has a different version in their PATH, lest things break.

Technically, the paths should be set by the build machinery. However, ARB is built using plain Makefiles with a bunch of tests and variables interspersed through those catering to Debian and Homebrew, not with cmake or autotols. I don't dare to mess with those too much, in particular not upstream, as it's too easy to break things under other conditions.